test/parallel/test-tls-connect.js

```diff
@@ -37,7 +37,7 @@ const path = require('path');
   const cert = fs.readFileSync(path.join(common.fixturesDir, 'test_cert.pem'));
   const key = fs.readFileSync(path.join(common.fixturesDir, 'test_key.pem'));
 
-  const options = {cert: cert, key: key, port: common.PORT};
```

Comment:
I'm not sure this should be changed, it may change the reason of the connection failure.